plugins/json-checkpoint/git_theta_json_checkpoint/checkpoints.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
do you mention this as an example anywhere? And why is it called git_theta_json_checkpoint - is the assumption that checkpoing-defining packages will be called git_theta_X_checkpoint?

Reply:
I added a mention in a new README for the plugin itself.

I also added a note about this, but plugins can be named whatever they want as long as they use the right string when listing their entry points. I just used git_theta_X_checkpoint as it seemed scoped and wouldn't collide with anything,